lib/bosh/workspace/project_deployment.rb

```diff
@@ -63,14 +54,34 @@ def execute_stub(stub_file)
 
     private
 
+    def stubs_dir
+      File.expand_path(File.join(file_dirname, "../stubs"))
+    end
+
+    def stub_file
+      Dir[File.join(stubs_dir, "#{file_basename('.*')}*")].first
```

Comment:
@allomov I'm fine with adding a `--stub-file` option however `bosh-workspaces` has always been about convention over configuration. With regards to just taking the first, it is not just taking the first. It is taking the first from the glob results which is using the file base name of the deployment file to filter results.